pkg/builder/webhook.go

```diff
@@ -251,3 +285,14 @@ func generateValidatePath(gvk schema.GroupVersionKind) string {
 	return "/validate-" + strings.ReplaceAll(gvk.Group, ".", "-") + "-" +
 		gvk.Version + "-" + strings.ToLower(gvk.Kind)
 }
+
+const webhookPathStringValidation = `^((/[a-zA-Z0-9-_]+)+|/)$`
+
+var validWebhookPathRegex = regexp.MustCompile(webhookPathStringValidation)
+
+func generateCustomPath(customPath string) (string, error) {
+	if !validWebhookPathRegex.MatchString(customPath) {
+		return "", errors.New("customPath \"" + customPath + "\" does not match this regex: " + webhookPathStringValidation)
+	}
+	return path.Clean(customPath), nil
```

Comment:
This alters the path or not? Lets not do that, people might refernce the path in their Validating/MutatingWebhookConfiguration

Reply:
@alvaroaleman if you're asking about the `path.Clean()` in the return section, it does not alter the path, it on makes it cleaners. For eg.

"//test" -> "/test"

So you want me to just return the `customPath`?

Reply:
> So you want me to just return the customPath?

I think so. I had a look at this and this is actually meant for filepaths, not for URLs. We are not dealing with untrusted input here, the double slash case you mentioned is probably confusing but not harmful. I think if for any reason we think the path we were given is wrong, we should return an error, otherwise the person who authors such code might not realize there is a problem

Reply:
Okay I understand! Just removed it!